ppl/src/main/java/com/amazon/opendistroforelasticsearch/sql/ppl/parser/AstBuilder.java

```diff
@@ -0,0 +1,177 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.ppl.parser;
+
+import com.amazon.opendistroforelasticsearch.sql.ppl.antlr.parser.OpenDistroPPLParser;
+import com.amazon.opendistroforelasticsearch.sql.ppl.antlr.parser.OpenDistroPPLParserBaseVisitor;
+import com.amazon.opendistroforelasticsearch.sql.ppl.plans.expression.Expression;
+import com.amazon.opendistroforelasticsearch.sql.ppl.plans.expression.Map;
+import com.amazon.opendistroforelasticsearch.sql.ppl.plans.logical.Aggregation;
+import com.amazon.opendistroforelasticsearch.sql.ppl.plans.logical.Filter;
+import com.amazon.opendistroforelasticsearch.sql.ppl.plans.logical.LogicalPlan;
+import com.amazon.opendistroforelasticsearch.sql.ppl.plans.logical.Project;
+import com.amazon.opendistroforelasticsearch.sql.ppl.plans.logical.Relation;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import org.antlr.v4.runtime.tree.ParseTree;
+
+@RequiredArgsConstructor
+public class AstBuilder extends OpenDistroPPLParserBaseVisitor<LogicalPlan> {
+    private final AstExpressionBuilder expressionBuilder;
+
+    @Override
+    public LogicalPlan visitPplStatement(OpenDistroPPLParser.PplStatementContext ctx) {
```

Comment:
Np: using static import can make this class cleaner.

Reply:
Yes that's what I would like to do but the IDEA was not able to automatically add them as the static methods. I will manually replace them